Generate docs even if it it has spaces

championshuttler opened the pull request:
Fixes #14

championshuttler committed:
[#4] Ability to install as python package from git

championshuttler committed:
add instruction to install through pip3

championshuttler committed:
Update README.md

championshuttler committed:
Readme addition to index.pdf

championshuttler committed:
Issue-14

Commit:
Fix the readme

Commit:
FIx dependencies

championshuttler committed:
Create LICENSE

championshuttler committed:
Fix. Unable to find ecldoc module due to file structure.

championshuttler committed:
Fix. gitignore.

lilyclemson committed:
Merge pull request #25 from championshuttler/license

Create LICENSE

lilyclemson commented on src/ecldoc/parseDoc.py:
```diff
@@ -9,6 +9,7 @@ def parseDocstring(docstring) :
     break into individual tags and
     return them as XML Elements
     '''
+    docstring = re.sub(r'([@])\s+', r'\1', docstring)
```
Please avoid capture patterns other than the line of @param

championshuttler committed:
[Issue-14]Fix. for only specific tags

lilyclemson reviewed (changes requested):
Still failed with same error: Empty tag value. Please let me know if anything is missing. 

P.S. Please pull request the following change to the branch I14-PR23

championshuttler committed:
pip3 standard installation (#19)

1. Update README.md
2. Fix. Unable to find ecldoc module due to file structure.
3. Fix. gitignore.
4. Remove textput log file
5. upgrade lxml version and readme

championshuttler committed:
Remove unwanted files

championshuttler committed:
Merge branch 'main' into issue-14

Commit:
Fix comments

championshuttler committed:
Update setup.py
